Neos.Neos/Classes/Fusion/NodeUriImplementation.php

```diff
@@ -129,11 +129,11 @@ public function evaluate()
         } else {
             return '';
         }
-        /** @todo implement us
+        /* TODO implement us see https://github.com/neos/neos-development-collection/issues/4524 {@see \Neos\Neos\ViewHelpers\Uri\NodeViewHelper::resolveNodeAddressFromString} for an example implementation
         elseif ($node === '~') {
         $nodeAddress = $this->nodeAddressFactory->createFromNode($node);
         $nodeAddress = $nodeAddress->withNodeAggregateId(
-        $this->nodeSiteResolvingService->findSiteNodeForNodeAddress($nodeAddress)->getNodeAggregateId()
+        $siteNode->nodeAggregateId
```

Comment:
Where does this come from?

Reply:
@mhsdesign doesn't your IDE scream at you?

Reply:
its a comment chill ^^

Reply:
https://github.com/neos/neos-development-collection/issues/4524